addons/website/static/src/js/menu/translate.js

```diff
@@ -147,7 +147,7 @@ var TranslatePageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         current.href = window.location.toString();
         current.search += (current.search ? '&' : '?') + 'enable_editor=1';
         // we are in translate mode, the pathname starts with '/<url_code/'
```

Comment:
The comment is wrong too and misleading, right?
Now, trailing / are removed so we actually are on `/<url_code>` (no trailing slash)

Reply:
It's right in all cases except for the homepage `/`. I'll add a note about the edge case.